lib/runner/extensions/event.command.js

```diff
@@ -331,6 +336,7 @@ module.exports = {
 
                     // Ensure that we have SDK instances, not serialized plain objects.
                     // @todo - should this be handled by the sandbox?
+                    result && result._variables && (result._variables = new sdk.VariableScope(result._variables));
```

Comment:
Shouldn't there be a stricter check on `result._variables?`

Reply:
VariableScope constructor handles weirdness in the parameters.